features/step_definitions/project_steps.rb

```diff
@@ -1,11 +1,20 @@
 Given(/^(\d+) projects?$/) do |count|
-  d.given_projects count: count
+  d.given_objects name: 'project', count: count
```

Comment:
I see you made the helper in the driver more generic taking the kind of thing to make.
It might be worth making the step generic as well and have a single step definition that covers both projects and tasks (and other future models)
`Given(/^(d+) (\w+)s?$/) do |count, kind|`